FEXCore/Source/Interface/IR/Passes/RedundantFlagCalculationElimination.cpp

```diff
@@ -17,54 +17,203 @@ desc: This is not used right now, possibly broken
 #include <array>
 #include <memory>
 
+// Flag bit flags
+#define FLAG_N (1 << 0)
```

Comment:
```suggestion
#define FLAG_N (1U << 0)
```

Maybe, just to enforce these as unsigned, since they're bit flags